### DISCUSSION TITLE
Lesson 11: NFT not displaying correctly

### DISCUSSION THREAD

**Discussion by SquilliamX** [Cyfrin/foundry-full-course-cu#713] [Q&A] [answered]
Hi everyone, attached is what my NFT looks like in my wallet. Can anyone please help me figure out why my nft looks like this? This is the basic nft

Here's my code - https://github.com/SquilliamX/Foundry-nft-f23

<img width="280" alt="Screenshot 2023-08-25 192717" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/134108406/4cceabf9-5401-43a6-8ada-a6d605218701">

<img width="292" alt="Screenshot 2023-08-25 192313" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/134108406/0ec71af6-1ff5-43b4-b5cd-fe2f384a5161">

**Comment by theirrationalone** [comment]
In your `Interactions.s.sol`, Please replace PUG value with these one of your shorten links:

`https://ipfs.io/ipfs/QmcJRiVTDEsFPWdUoMWMUr8R9VaqqWd64tWJahJh49JfZy` -> Supports `https` protocol.
`ipfs.io/ipfs/QmcJRiVTDEsFPWdUoMWMUr8R9VaqqWd64tWJahJh49JfZy` -> Supports `ipfs` protocol.

and then try Again.... if not work then, Please try out this [discussion](https://github.com/Cyfrin/foundry-full-course-f23/discussions/536)

**Comment by SquilliamX** [comment, marked as answer]
The problem was my indeed my ``Interactions.s.sol``, but it was fixed when i changed it to the ipfs links from the Patrick's foundry-nft repo here: https://github.com/Cyfrin/foundry-nft-f23/blob/main/script/Interactions.s.sol
